debugfs/dump.c

```diff
@@ -392,9 +418,14 @@ void do_rdump(int argc, char **argv, int sci_idx,
 		else
 			basename = arg;
 
-		if (rdump_inode(ino, &inode, basename, dest_dir) != 0)
+		if (rdump_inode(ino, &inode, basename, dest_dir, preserve) != 0)
 			ss_set_exit_status(sci_idx, 1);
 	}
+	print_usage:
+		com_err(argv[0], 0,
+				"Usage: rdump [-p] "
+				"<fs_directory>... <output_file>");
+		ss_set_exit_status(sci_idx, 1);
```

Comment:
A `return` should be added before the label.
Also, two minor changes: indent should be decreased one level, and output_file changed to output_directory.

```suggestion
	return;

print_usage:
	com_err(argv[0], 0,
			"Usage: rdump [-p] "
			"<fs_directory>... <output_directory>");
	ss_set_exit_status(sci_idx, 1);
```